MPSGameLift/Documentation/GameLift.md

```diff
@@ -106,14 +83,15 @@ aws gamelift create-fleet --region us-west-2 --name GameLiftO3DTest2016 --ec2-in
 **NOTE**
 
 The ec2-instance-type and fleet-type determines the kind of AWS resources used; your AWS account may incur costs.
+https://aws.amazon.com/ec2/pricing/reserved-instances/pricing/
```

Comment:
If this is running on GameLift, then link to the GameLift pricing documentation?